include/picongpu/particles/bremsstrahlung/Bremsstrahlung.tpp

```diff
@@ -149,7 +150,7 @@ float3_X Bremsstrahlung<T_IonSpecies, T_ElectronSpecies, T_PhotonSpecies>::scatt
     float_X sinTheta, cosTheta;
     math::sincos(theta, sinTheta, cosTheta);
 
-    const float_X phi = -float_X(M_PI) + float_X(2.0) * float_X(M_PI) * this->randomGen(acc);
+    const float_X phi = -math::Pi<float_X>::value + math::Pi<float_X>::doubleValue * this->randomGen(acc);
```

Comment:
CODING STYLE:
The template argument should be surrounded by spaces: `math::PI< float_X >::...`
see: https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#17-template-instantiations

Reply:
see comment of @ax3l - this only applies to new files

Reply:
if you like, you can "dismiss" your own review at the bottom

Reply:
Since this style issue was my only remark, I approved the pull request.

Reply:
Yeah, I tried to match the brackets and spacing to the existing style in each file. However, ofc I might have still missed smth. And also a couple of files used mixed style, so I preferred a new style in those.

Reply:
That's correct: try match existing style in old files (as long as reasonable) for small changes and go directly for new style in new files.